File: namsor/typescript-node/src/JapaneseNameLatinCandidatesExample.ts
import * as fs from 'fs';
import api from "namsor_client"
import models from "namsor_client"

const apiCaller = new api.JapaneseApi();
apiCaller.setApiKey(api.JapaneseApiApiKeys.api_key, "YOUR_API_KEY");

apiCaller.japaneseNameLatinCandidates(
  "塩田", // japaneseSurnameKanji
  "千春", // japaneseGivenNameKanji
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling JapaneseApi#japaneseNameLatinCandidates:");
  console.log(error.body);
});

File: namsor/kotlin/src/main/kotlin/oseg/namsor_examples/JapaneseNameLatinCandidatesExample.kt
package oseg.namsor_examples

import app.namsor.client.infrastructure.*
import app.namsor.client.apis.*
import app.namsor.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class JapaneseNameLatinCandidatesExample
{
    fun japaneseNameLatinCandidates()
    {
        ApiClient.apiKey["X-API-KEY"] = "YOUR_API_KEY"

        try
        {
            val response = JapaneseApi().japaneseNameLatinCandidates(
                japaneseSurnameKanji = "塩田",
                japaneseGivenNameKanji = "千春",
            )

            println(response)
        } catch (e: ClientException) {
            println("4xx response calling JapaneseApi#japaneseNameLatinCandidates")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling JapaneseApi#japaneseNameLatinCandidates")
            e.printStackTrace()
        }
    }
}
